File: printing_test/printing_test.py
from datetime import datetime

def ts():
    return datetime.now().timestamp()

def execution_time(start: datetime.timestamp, end: datetime.timestamp) -> None:
    print(f"{round((end - start) * 1000, 3)} ms")

if __name__ == "__main__":
    start = ts()

    for _ in range(10**4):
        print("Hello World!")

    end = ts()
    execution_time(start, end)

File: printing_test/printing_test.cpp
#include <iostream>
#include <time.h>

int main() {
    clock_t start = clock();

    for (int i = 0; i < 10000; i++) {
        std::cout << "Hello World!" << std::endl;
    }

    clock_t end = clock();

    std::cout << (double)(end - start) / CLOCKS_PER_SEC * 1000 << " ms" << std::endl;

    return 0;
}